src/internal/modules/billing/controller.js

```diff
@@ -299,6 +299,31 @@ const getTransactionsCSV = async (request, h) => {
     .header('Content-disposition', `attachment; filename="${fileName}"`);
 };
 
+/**
+ * Remove an invoice from the bill run
+ * @param {*} request
+ * @param {*} h
+ */
+const getBillingBatchDeleteAccount = async (request, h) => {
+  const batchId = request.params.batchId;
+  const invoiceId = request.params.invoiceId;
+  const account = await batchService.getBatchInvoice(batchId, invoiceId);
+  return h.view('nunjucks/billing/batch-delete-account', {
+    ...request.view,
+    pageTitle: 'You are about to remove this invoice from the bill run',
+    account,
+    batch: { id: request.params.batchId },
+    back: `/billing/batch/${batchId}/summary`
+  });
+};
+
+const postBillingBatchDeleteAccount = async (request, h) => {
+  const accountId = request.params.invoiceId;
+  const batchId = request.params.batchId;
+  await services.water.billingBatchService.deleteAccountFromBatch(batchId, accountId);
+  return h.redirect(`/billing/batch/${request.params.batchId}/summary`);
```

Comment:
It looks like you missed this function when you were writing your tests

Reply:
done and i added some unit tests for the other post methods that also didn't have any tests